google-cloud-spanner/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+### 1.13.0 / 2019-11-06
+
+#### Features
+
+* Add InstanceConfig#replicas (ReplicaInfo)
```

Comment:
This is the low-level API. Should we update the description of this change?

Reply:
Yes. The version bump also needs to be reduced to patch.